carbon-design-system/carbon
# fix: fixed the `aria-label` assigning to wrapper div of dropdown

Closes #17558 

`aria-label` value, which was getting assigned to the wrapper div which encircles the dropdown, was caused by downshift's change, is fixed.

#### Changelog

**New**

- Nothing's new

**Changed**

- passing `aria-label` prop to the downshift's `getToggleButtonProps()` 

**Removed**

- removed `aria-label` prop that has been passed to `ListBox`

#### Testing / Reviewing

- Navigate to Dropdown component's playground story from deploy preview
- In the controls section, add `aria-label` as "test" or anything of you liking
- Run the IBM's accessibility checker extension after going to full view mode on the component
- The accessibility violations mentioned in the parent [issue](https://github.com/carbon-design-system/carbon/issues/17558#:~:text=No%20response-,Description,-When%20titleText%20isn%27t) should not persist.
